actionpack/test/dispatch/debug_exceptions_test.rb

```diff
@@ -89,7 +89,7 @@ def call(env)
         begin
           raise ActionController::ParameterMissing, :missing_param_key
         rescue
-          raise NameError.new("uninitialized constant Userr")
+          raise NameError.new("uninitialized constant User")
```

Comment:
This was by design. We want Userr here to how there was a type in the constant name